protractor.conf.js

```diff
@@ -24,7 +24,7 @@ const SAVE_SCREENSHOT = process.env.SAVE_SCREENSHOT === 'true';
 const API_CONTENT_HOST = process.env.API_CONTENT_HOST || 'http://localhost:8080';
 const MAXINSTANCES = process.env.MAXINSTANCES || 1;
 const MAX_RETRIES = process.env.MAX_RETRIES || 1;
-const LOG_LEVEL = process.env.LOG_LEVEL || 'ERROR';
```

Comment:
better to not change this variable is the same in all he repo and documented

Reply:
It's E2E_LOG_LEVEL in apps and adw and LOG_LEVEL in adf.... :)
I don't know who to be consistent with :D

Reply:
@eromano , please see my previous comment and let me know if you still think I should keep the name LOG_LEVEL.

Reply:
then can we cahnge it aso in ADF and change it also in the confluence page?https://alfresco.atlassian.net/wiki/spaces/AP/pages/471169/Running+e2e+tests